packages/riverpod_annotation/lib/riverpod_annotation.dart

```diff
@@ -25,6 +25,13 @@ export 'package:riverpod/src/internals.dart'
         $RefArg,
         $ProviderPointer,
 
+        // Mutation/Listenables
+        ProviderListenable,
+        LazyProxyListenable,
```

Comment:
Those aren't public APIs. We should rename those we $ if we want to show them to the generator

Reply:
$LazyProxyListenable seems fine, but ProviderListenable looks like its used in a lot of places that seems like it needs to be resolvable publicly, and I don't have enough experience to tell whether they can simply all be safely renamed. Your thoughts?